### ISSUE TITLE
Visibility on releases, release notes and CI CD pipelines

### ISSUE THREAD

**Issue by baywet** [Azure/azure-functions-microsoftgraph-extension#78]
This repo lacks of the following things:
- Github release every time the nuget packages are published. This will help having consistent release notes, tracking whether changes are available or not, tracking PR status…
- Visibility on the CI CD pipeline (master triggers production packages, dev triggers beta packages?)
- CD pipeline (doesn't seem in place today) to have automated publishing of the packages, creation of the releases...

**Comment by ConnorMcMahon** [comment]
@baywet, you are right. We plan on overhauling a lot of this in the near future, and when we start putting more resources to these bindings, we will definitely put more efforts on improving the visibility of these changes.
